src/pages/survey/[surveyId]/index.tsx

```diff
@@ -30,15 +31,19 @@ function AnswerPage({
     activeQuestionIndex,
     handleNextQuestion,
     handlePreviousQuestion,
+    QuestionLength,
   } = useSurveyAnswerManager(initialData);
-  const { t } = useTranslation('survey');
 
+  const { t } = useTranslation('survey');
   return (
     <>
       <Head>
         <title>{t('title')}</title>
         <meta name="description" content={t('content')} />
       </Head>
+      <div>
+        <Progressbar currentStep={activeQuestionIndex} totalSteps={QuestionLength} className="w-full" />
```

Comment:
we don't need pass `w-full` here, since I see that is applied in `Progressbar` component.

Reply:
In addition, please place that component under questions

Reply:
Sure, I understand!